packages/bundle/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@babel/runtime": "7.12.5",
     "adaptivecards": "2.5.0",
-    "botframework-directlinejs": "0.13.1",
+    "botframework-directlinejs": "0.14.1",
```

Comment:
Doesn't this need to be 14.2?

Reply:
0.14.1 is correct, it includes both @timenick and @stevengum PR. 😉